proto/tendermint/abci/types.proto

```diff
@@ -133,6 +133,8 @@ message RequestPrepareProposal {
   tendermint.types.Data block_data = 1;
   // If an application decides to populate block_data with extra information, they can not exceed this value.
   int64 block_data_size = 2;
+  // proposal_header is the header from the proposal
```

Comment:
[question] is this godoc copied from somewhere? If not, proposal to make the godoc reference the field name b/c I'm not sure what the relationship is between `proposal_header` and `chain_id` but they seem distinct. 

```suggestion
  // chain_id is a unique identifier for the blockchain network this proposal belongs to (i.e. mocha-1)
```

Reply:
I'll create an issue for this one, as changing will require recompiling the proto and the CI and my setup and different somehow despite using the same versions...

Reply:
#1029